### PR TITLE
solved chapter1 - exercise 2

### PR DESCRIPTION
**Pull request by ghost**
Did all these steps:

Add element at the start of an array
    Add element at the end of an array
    Add element at any position of an array
    Remove element at the start of an array
    Remove element at the end of an array
    Remove element at a given index of an array
    Login: takes username and password. If they match a hidden username and password- a message "Logged in!" is printed. Username capitalization does not matter, password capitalization must match.

### CODE REVIEW

File: Chapter1/Array_Functions.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        {
+            return Remove_Nth_Place_Array(array, array.Length -1 ); 
+        }
+
+        public static int[] Remove_Nth_Place_Array(int[] array, int index)
+        {
+            var new_array = new int[array.Length - 1];  
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (i == index)
```
Comment (aduckinpants) [inline Chapter1/Array_Functions.cs:79]: Would the same trick used in `Add_Nth_Place_Array` allow you to reduce the branch comparisons in the loop?

**Review by aduckinpants — APPROVED** [review]

File: Chapter1/Array_Functions.cs (hunk 1 above)
Comment (ghost) [reply Chapter1/Array_Functions.cs:79]: yep I could . Will try it

File: Chapter1/Array_Functions.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
+﻿using System;
+
+namespace BootCamp.Chapter1
+{
+    public static class Array_Functions
+    {
+        public static int[] Sort_Array(int[] array)
```
Comment (Almantask) [inline Chapter1/Array_Functions.cs:7]: In C#, don't use snake_case.

Comment (Almantask) [reply Chapter1/Array_Functions.cs:7]: Use PascalCase

**Review by Almantask — APPROVED** [review]
Regardless of bad naming conventions, looks good. Not sure about the value of comments too, they seem to be explaining obvious stuff.

**Pull request closed without merging**